demo/gnmi_telemetry/gnmi.go

```diff
@@ -104,15 +105,30 @@ func CreateAFTInstance() (*oc.Device, error) {
 	// Each union has a method that is named To_X where the X is the union type, associated
 	// with the struct that the union is within. This attempts to return the right type
 	// based on the input interface.
-	expNull, err := nh.To_NetworkInstance_Afts_LabelEntry_NextHop_PushedMplsLabelStack_Union(oc.OpenconfigAft_NextHop_PushedMplsLabelStack_IPV4_EXPLICIT_NULL)
+	expNull, err := nh.To_NetworkInstance_Afts_NextHop_PushedMplsLabelStack_Union(oc.OpenconfigAft_NextHop_PushedMplsLabelStack_IPV4_EXPLICIT_NULL)
 	if err != nil {
 		return nil, fmt.Errorf("error converting explicit null to union, got: %v", err)
 	}
-	nh.PushedMplsLabelStack = []oc.NetworkInstance_Afts_LabelEntry_NextHop_PushedMplsLabelStack_Union{
-		&oc.NetworkInstance_Afts_LabelEntry_NextHop_PushedMplsLabelStack_Union_Uint32{42},
-		&oc.NetworkInstance_Afts_LabelEntry_NextHop_PushedMplsLabelStack_Union_Uint32{84},
+	nh.PushedMplsLabelStack = []oc.NetworkInstance_Afts_NextHop_PushedMplsLabelStack_Union{
+		&oc.NetworkInstance_Afts_NextHop_PushedMplsLabelStack_Union_Uint32{42},
+		&oc.NetworkInstance_Afts_NextHop_PushedMplsLabelStack_Union_Uint32{84},
 		expNull,
 	}
 
 	return d, nil
 }
+
+// addUnaryNextHop adds a next-hop-group and next-hop with the specified ID to the
```

Comment:
Are these changes related to the leaf getters?

Reply:
These changes are made because the schema changed for the public OpenConfig, and to demo the leaf getters I regenerated our example schema -- which then mandated me to make these changes. I can revert the schema tests and make this separate commits if you'd prefer.

Reply:
Understood, that is fine, thanks for explaining.